repair/row_level.cc

```diff
@@ -2486,10 +2487,13 @@ class row_level_repair {
                     send_missing_rows_to_follower_nodes(master);
                 }
             } catch (std::exception& e) {
+                std::string err(e.what());
+                if (err.find("Can't find a column family") == 0) {
```

Comment:
This is extermely clunky. Let's find a better way to communicate this.

Reply:
I do not like this neither. Unfortunately, RPC losts the exception type info.  The proper fix would be in rpc layter.  @gleb-cloudius what do you think?

Reply:
@asias - we discussed this a bit - we do not want to extend rpc for 4.0 but would rather have the server side and return as a normal return and not as an exception - can this be done ?

Reply:
I think it is even more ugly than comparing the error msg in the exception. We need to patch all the verbs that might return the can't find a column family exception. Some of verbs return nothing but a future<>, there is no way to extend such rpc verb to send back the error message.  We have verbs that returns future<uint64_t>, no way to extend it in idl.

Reply:
How about creating new verbs with better returns and using them if they exist?

Reply:
This is pushing the job of the rpc framework to the users of rpc. Our rpc is pretty hard to use already, it is better not to make it even harder. It is not better returns, it is worse returns with an error code. We need to change all the verbs. We need to figure out if the new verb is supported. IMO, it does not deserve all the extra complexity introduced because we want to delay the fix at the rpc layer. 

Alternatively, we can just fail the repair for both normal repair and bootstrap operations if a table is dropped. We document do not change the schema during a node operation.

Reply:
We recommend users not to have DDL changes but can't really force that, some systems have this done programaticlly on the user end. Adding node is a long running operation, there is no way to grantee their will not be DDL.